src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/CreateReindexRequestHandler.cs

```diff
@@ -92,23 +87,10 @@ public async Task<CreateReindexResponse> Handle(CreateReindexRequest request, Ca
 
                     searchParameterResourceTypes.UnionWith(searchParameterInfo.BaseResourceTypes);
                 }
```

Comment:
This was removed because adding a guid as an indicator for the search param hash was incorrect to begin with and wasn't noticed until doing this work.